ui/components-react/package.json

```diff
@@ -3,24 +3,19 @@
   "version": "5.0.0-dev.0",
   "description": "A react component library of iTwin.js UI data-oriented components",
   "type": "module",
-  "main": "lib/cjs/components-react.js",
-  "module": "lib/esm/components-react.js",
-  "types": "lib/cjs/components-react",
   "exports": {
-    ".": {
-      "import": "./lib/esm/components-react.js",
-      "require": "./lib/cjs/components-react.js"
+    ".": "./lib/components-react.js",
+    "./internal": "./lib/internal.js",
+    "./package.json": "./package.json",
+    "./lib/cjs/*": {
```

Comment:
Why `/lib/cjs` entry is left for sass files?

Reply:
Previously consumers would import SCSS files via these paths and I didn't want to force everyone to upgrade at once (I will leave a recommendation in NextVersion.md to get rid of the SASS imports instead)

Reply:
I.e. `@import "@itwin/core-react/lib/cjs/core-react/style/themecolors.scss"` continues to work and resolves to `@import "@itwin/core-react/lib/core-react/style/themecolors.scss"`